### PR TITLE
Update total supply of CUB & BLUE

### PR DESCRIPTION
**Pull request by lioncubdao**

### CODE REVIEW

File: terra/asset.json [hunk 1]
```diff
@@ -1133,6 +1133,7 @@
     "decimals": "6",
     "type": "cw20",
     "circ_supply_api": "https://raw.githubusercontent.com/lioncubdao/logo/main/blue_supply",
+    "total_supply_api": "fromStaticAmount",
```
Comment (yusufaine) [inline terra/asset.json:1136]: what's the total supply amount for BLUE?

File: terra/asset.json [hunk 2]
```diff
@@ -1143,6 +1144,7 @@
     "decimals": "6",
     "type": "cw20",
     "circ_supply_api": "https://raw.githubusercontent.com/lioncubdao/logo/main/cub_supply",
+    "total_supply_api": "fromStaticAmount",
```
Comment (yusufaine) [inline terra/asset.json:1147]: Similarly, what's the total supply amount for CUB?

**Review by yusufaine — COMMENTED** [review]
No change is needed for the PR, but we would need the static amounts for the total supply after the division by the decimal amount for BLUE and CUB so that we could update said values in the database.

File: terra/asset.json (hunk 1 above)
Comment (lioncubdao) [reply terra/asset.json:1136]: Thank you! the token supply of CUB is one trillion (1000000000000)

File: terra/asset.json (hunk 2 above)
Comment (lioncubdao) [reply terra/asset.json:1147]: the token supply of CUB is one trillion (1000000000000)

**Pull request merged**